README.hebrew.md

```diff
@@ -942,13 +942,13 @@ API, הודעה חדשה נרשמת לתור, וקריאה לכלי צפיה ב
 
 <br/><br/>
 
-## ![✔] 5.13. Use tools that automatically detect vulnerabilities
+## ![✔] 5.13. השתמשו בכלים שמזהים פגיעויות אוטומטית
 
-**אמ;לק:** Even the most reputable dependencies such as Express have known vulnerabilities (from time to time) that can put a system at risk. This can be easily be tamed using community and commercial tools that constantly check for vulnerabilities and warn (locally or at GitHub), some can even patch them immediately
+**אמ;לק:** גם לתלויות המכובדות ביותר כמו למשל Express יש פגיעויות ידועות (מפעם לפעם) שיכולות לסכן את המערכת שלכם. ניתן להתמודד עם כך בקלות ע״י שימוש בכלים קהילתיים או מסחריים שבודקים בקביעות הימצאות פגיעויות ולהזהיר (מקומית או בגיטהב), חלקם יכולים אפילו לתקן אותם מידית.
 
-**אחרת:** Keeping your code clean from vulnerabilities without dedicated tools will require you to constantly follow online publications about new threats. Quite tedious
+**אחרת:** שמירת הקוד שלכם נקי מפגיעויות ללא שימוש בכלים יעודיים יצריך מכם לעקוב בקביעות אחר פרסומים על איומים חדשים ברשת. די מתיש. 
 
-🔗 [**Read More: Use tools that automatically detect vulnerabilities**](./sections/production/detectvulnerabilities.md)
+🔗 [**קרא עוד: השתמשו בכלים שמזהים פגיעויות אוטומטית**](./sections/production/detectvulnerabilities.md)

```

Comment:
@ShaiDemri Thanks for take a part on the Hebrew translation.

pls change the `קרא עוד` to `לקריאה נוספת`

great work